frontend/redux/selectors.js

```diff
@@ -24,3 +24,22 @@ export const isLoading = state => Object
 
 export const demographics = state => state.demographics.questions;
 export const demographicsOrder = state => state.demographics.order;
+
+export const selectWorkload = function(state) {
+  const images = [];
+  const answers = state.demographics.answers;
+
+  Object.keys(answers).map((i) => {
```

Comment:
lint is complaining because you're using `map` without using the return value.  Use `forEach` or maybe actually just:

```
const images = Object.keys(answers).map( id => ({
  id: Number(id),
  demographics: answers[i]
}));
```